keras/layers/containers.py

```diff
@@ -20,11 +20,6 @@ class Sequential(Layer):
 
     def __init__(self, layers=[]):
         self.layers = []
-        self.params = []
```

Comment:
Why are these attributes being remove? It causes tests to fail, and they are definitely necessary.